tasks/e2e-simple.sh

```diff
@@ -80,7 +80,7 @@ cd "$root_path"
 nodeVersion=`node --version | cut -d v -f2`
 nodeMajor=`echo $nodeVersion | cut -d. -f1`
 nodeMinor=`echo $nodeVersion | cut -d. -f2`
-if [[ nodeMajor -lt 4 ]] || [[ nodeMajor -eq 4  &&  nodeMinor -lt 7 ]]
+if [[ nodeMajor -lt 6 ]]
```

Comment:
I've removed the Node 4 tasks from `travis.yml`. I also removed them from `appveyor.yml`.